### DISCUSSION TITLE
How can I clean up this log?

### DISCUSSION THREAD

**Discussion by frankyw** [Flexget/Flexget#2820] [Q&A]
Hi all,

I am interested in cleaning up my log file and would love to get opinions or suggestions on the below. This is [my config file](https://github.com/frankyw/flexget-config/blob/master/config.yml).

I am interested in whether there is a way to actually "fix" these "errors" by changing my config. Or perhaps it would be great to have a log level setting per plugin, or maybe tweaking the log levels of some of these events, as there is quite a bit of unnecessary "noise" in the log.

What I'm actually interested in is when there are real errors, and when something downloads - basic info about what is happening. Thanks!

<img width="1139" alt="Screen Shot 2021-01-23 at 8 34 09 AM" src="https://user-images.githubusercontent.com/2604571/105579669-ceea5880-5d55-11eb-8a3b-6c8367614986.png">

**Multiple IMDB lookup failures**

I am basically seeing the same message twice each time a new file on IRC matches a movie regex, but is not found when doing a lookup on IMDB. I don't really care about seeing this, especially twice, each time something is not found on IMDB - when the purpose is to match the file to something in the movie list. Am I using this plugin incorrectly?

**Multiple api_bluray failures**

Each time flexget searches for movies, I see lots of errors for things not found on blu-ray.com that I don't care about. If these are failures from movies that do not yet exist on blue-ray.com, should this really be considered an error? I would say this is even a VERBOSE level event, even if I want basic INFO, I probably don't care about this.

**Comment by frankyw** [comment]
I have tried the [log_filter](https://github.com/jawilson/flexget-config/blob/master/flexget/plugins/log_filter.py) plugin but it does not seem to filter anything, despite being activated and applying the filter on the correct context.

```
log_filter:
  imdb_lookup:
    - lookup failed
  tmdb_lookup:
    - TMDB lookup failed
  api_bluray:
    - Error refreshing movie details
```

**Comment by gazpachoking** [comment]
That log_filter plugin isn't compatible with the latest version of FlexGet. I have an updated version here https://github.com/Flexget/Flexget/pull/2740, but there are some outstanding questions, including whether it should be included at all.

**Reply by frankyw** [reply]
Thanks, I have a left a comment on the PR. I understand changes to the core are necessary for this to work again, so hope it will be resolved soon.  I don't see any reason why the logger would not be callable by a plugin like this, as it was before.
